src/compiler/visitor.ts

```diff
@@ -1466,7 +1466,9 @@ namespace ts {
      * Aggregates the TransformFlags for a Node and its subtree.
      */
     export function aggregateTransformFlags<T extends Node>(node: T): T {
+        const stickyFlags = node.transformFlags & TransformFlags.StickyFlags;

```

Comment:
I don't think this is a good approach. `NeverApplyImportHelper` should be on `EmitFlags`.